src/dao/SituationFeedbackDAO.ts

```diff
@@ -49,6 +49,21 @@ export class SituationFeedbackDAO extends BaseDAO {
     });
   }
 
+  public async getTags(prefix: string): Promise<string[]> {
+    const options = new OnmsHTTPOptions();
```

Comment:
Even though you don't need anything specific option-wise here, I'd recommend still using `this.getOptions()` since that is the place we would implement things that should automatically apply, ie, if we decide in the future we need a certain header to be set for version > X.

Reply:
oop, never mind, that would require being a getter-type DAO...  I rescind my comment  ;)